package.json

```diff
@@ -21,6 +21,7 @@
   },
   "scripts": {
     "build": "lerna run build",
+    "build-docs": "typedoc packages/stream/index.ts",
```

Comment:
This should be in packages/stream/package.json

Also, append it to prepublishOnly command:

"prepublishOnly": "npm run lint && npm test && npm run build"
->
"prepublishOnly": "npm run lint && npm test && npm run build && npm run build-docs"